src/Service/Trigger/Recipient/RecipientCollection.php

```diff
@@ -13,4 +13,16 @@ public function offsetSet($index, $newval)
 
         parent::offsetSet($index, $newval);
     }
-}
\ No newline at end of file
+
+    public function jsonSerializer()
```

Comment:
```suggestion
    public function jsonSerialize()
```

https://www.php.net/manual/fr/jsonserializable.jsonserialize.php